tutorials/nservicebus-101/lesson-2/Solution/ClientUI/Program.cs

```diff
@@ -18,7 +18,9 @@ static void Main()
             Console.Title = "ClientUI";
 
             var endpointConfiguration = new EndpointConfiguration("ClientUI");
-            endpointConfiguration.UseTransport<MsmqTransport>();
+
+            var transport = endpointConfiguration.UseTransport<MsmqTransport>();
```

Comment:
The unused transport variable makes things a lot easier when we start adding routing, or immediately if they select the SQL transport. I'll make sure the text points this out.